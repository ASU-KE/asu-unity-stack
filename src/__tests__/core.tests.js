
// React needs to be in scope for JSX to work
import React from 'react';
import { create as renderer } from 'react-test-renderer';
import createContext from 'react-cosmos-test/generic';
import fixture from '../__fixtures__/core.fixtures';

const { mount, getWrapper } = createContext({
    renderer,
    fixture: fixture[0]
});

beforeEach(mount);

test('matches snapshot', () => {
    // Careful, this is no longer an Enzyme wrapper, but a react-test-renderer wrapper!
    expect(getWrapper().toJSON()).toMatchSnapshot();
});