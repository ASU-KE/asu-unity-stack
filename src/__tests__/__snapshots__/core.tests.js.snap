// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 2`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 3`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 4`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 5`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 6`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 7`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 8`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 9`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 10`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 11`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 12`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 13`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 14`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 15`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;

exports[`matches snapshot 16`] = `
<div>
  <button
    className="btn btnBlue btnLg"
    label="Large explore Programs Button"
  >
    Large explore Programs Button
  </button>
</div>
`;
